src/wcs1_0/src/test/java/org/geoserver/wcs/GetCapabilitiesTest.java

```diff
@@ -391,4 +395,38 @@ public void testTimeRangeCoverage() throws Exception {
         assertXpathEvaluatesTo("2008-10-31T00:00:00.000Z", base + "/gml:timePosition[1]", dom);
         assertXpathEvaluatesTo("2008-11-07T00:00:00.000Z", base + "/gml:timePosition[2]", dom);
     }
+
+    protected void setupFixedValueRasterDimension(
+            QName layer,
+            String metadata,
+            DimensionPresentation presentation,
+            Double resolution,
+            String fixedValues) {
+        CoverageInfo info = getCatalog().getCoverageByName(layer.getLocalPart());
+        DimensionInfo di = new DimensionInfoImpl();
+        di.setEnabled(true);
+        di.setFixedValues(fixedValues);
+        di.setPresentation(presentation);
+        if (resolution != null) {
+            di.setResolution(BigDecimal.valueOf(resolution));
+        }
+        info.getMetadata().put(metadata, di);
+        getCatalog().save(info);
+    }
+
+    @Test
+    public void testFixedValueTimeRasterCoverage() throws Exception {
+        String fixedValueTime = "2008-11-09T00:00:00.000Z";
```

Comment:
Weak testing, we need some tests with some more values, also using different presentation modes.